config_defaults_inc.php

```diff
@@ -4140,6 +4140,8 @@
  * - DISPLAY_ERROR_INLINE  Display a one line error and continue execution.
  * - DISPLAY_ERROR_NONE    Suppress the error (no display). This is the default
  *                         behavior for unspecified errors constants.
+ * - DISPLAY_ERROR_HEADERS Stop and return an HTTP error header. No output
```

Comment:
See install.php for example where it sets up expected behaviors for error handling, timeout, and maintenance mode.

    error_reporting( E_ALL );
    @set_time_limit( 0 );

    # Load the MantisDB core in maintenance mode. This mode will assume that
    # config/config_inc.php hasn't been specified. Thus the database will not be opened
    # and plugins will not be loaded.
    define( 'MANTIS_MAINTENANCE_MODE', true );